conmon/conmon.c

```diff
@@ -475,8 +476,52 @@ int main(int argc, char *argv[])
 		errno = old_errno;
 		pexit("Failed to wait for `runtime %s`", exec ? "exec" : "create");
 	}
-	if (!WIFEXITED(runtime_status) || WEXITSTATUS(runtime_status) != 0)
+	if (!WIFEXITED(runtime_status) || WEXITSTATUS(runtime_status) != 0) {
+		if (sync_pipe_fd > 0 && !exec) {
+			if (terminal) {
+				/* 
+				 * The stderr is captured in the parent when terminal is passed down.
+				 * Just send pid as -1 to indicate create container issue to parent.
+				 */
+				len = snprintf(buf, BUF_SIZE, "{\"pid\": %d}\n", -1);
```

Comment:
nm, it's described in the above comment...

`Just send pid as -1 to indicate create container issue to parent.`